dbt_subprojects/dex/models/trades/base/platforms/swapline_base_base_trades.sql

```diff
@@ -0,0 +1,50 @@
+{{ config(
+    schema = 'swapline_base',
+    alias = 'base_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
+) }}
+
+WITH token_swaps AS (
+    SELECT
+        evt_block_number AS block_number,
+        CAST(evt_block_time AS timestamp(3) with time zone) AS block_time,
+        evt_tx_from AS maker,
+        evt_tx_to AS taker,
+        amount0In AS token_sold_amount_raw,
+        amount0Out AS token_bought_amount_raw,
+        sender AS token_sold_address,
+        to AS token_bought_address,
+        contract_address AS project_contract_address,
+        evt_tx_hash AS tx_hash,
+        evt_index AS evt_index
+    FROM
+        {{ source('swapline_base', 'SwaplinePair_evt_Swap') }}
+    {% if is_incremental() %}
+    WHERE
+        {{ incremental_predicate('evt_block_time') }}
+    {% endif %}
+)
```

Comment:
Maybe we need to apply `uniswap_compatible_v2_trades` macro here with `LBFactory_evt_LBPairCreated` table to produce dex.trades.